browser_tests/tests/bottomPanel.spec.ts

```
@@ -0,0 +1,105 @@
+import {
```

ESLint

[error] Resolve error: File 'astro/tsconfigs/strict' not found.
    at resolveExtends (file:///home/jailuser/git/node_modules/.pnpm/get-tsconfig@4.13.7/node_modules/get-tsconfig/dist/index.mjs:7:7238)
    at _parseTsconfig (file:///home/jailuser/git/node_modules/.pnpm/get-tsconfig@4.13.7/node_modules/get-tsconfig/dist/index.mjs:7:8131)
    at parseTsconfig (file:///home/jailuser/git/node_modules/.pnpm/get-tsconfig@4.13.7/node_modules/get-tsconfig/dist/index.mjs:7:13103)
    at resolve (file:///home/jailuser/git/node_modules/.pnpm/eslint-import-resolver-typescript@4.4.4_eslint-plugin-import-x@4.16.1_@typescript-eslin_ed9f55b13893a539f14f3a3aedd0ada3/node_modules/eslint-import-resolver-typescript/lib/index.js:66:67)
    at Object.resolve (file:///home/jailuser/git/node_modules/.pnpm/eslint-import-resolver-typescript@4.4.4_eslint-plugin-import-x@4.16.1_@typescript-eslin_ed9f55b13893a539f14f3a3aedd0ada3/node_modules/eslint-import-resolver-typescript/lib/index.js:147:20)
    at file:///home/jailuser/git/node_modules/.pnpm/eslint-plugin-import-x@4.16.1_@typescript-eslint+utils@8.56.0_eslint@9.39.1_jiti@2.6.1__ef4159fe4f412a19f929f063c6fe191d/node_modules/eslint-plugin-import-x/lib/utils/resolve.js:170:69
    at setRuleContext (/home/jailuser/git/node_modules/.pnpm/eslint-import-context@0.1.9_unrs-resolver@1.11.1/node_modules/eslint-import-context/lib/index.js:23:20)
    at fullResolve (file:///home/jailuser/git/node_modules/.pnpm/eslint-plugin-import-x@4.16.1_@typescript-eslint+utils@8.56.0_eslint@9.39.1_jiti@2.6.1__ef4159fe4f412a19f929f063c6fe191d/node_modules/eslint-plugin-import-x/lib/utils/resolve.js:170:30)
    at relative (file:///home/jailuser/git/node_modules/.pnpm/eslint-plugin-import-x@4.16.1_@typescript-eslint+utils@8.56.0_eslint@9.39.1_jiti@2.6.1__ef4159fe4f412a19f929f063c6fe191d/node_modules/eslint-plugin-import-x/lib/utils/resolve.js:215:12)
    at resolve (file:///home/jailuser/git/node_modules/.pnpm/eslint-plugin-import-x@4.16.1_@typescript-eslint+utils@8.56.0_eslint@9.39.1_jiti@2.6.1__ef4159fe4f412a19f929f063c6fe191d/node_modules/eslint-plugin-import-x/lib/utils/resolve.js:220:16)
    at checkSourceValue (file:///home/jailuser/git/node_modules/.pnpm/eslint-plugin-import-x@4.16.1_@typescript-eslint+utils@8.56.0_eslint@9.39.1_jiti@2.6.1__ef4159fe4f412a19f929f063c6fe191d/node_modules/eslint-plugin-import-x/lib/rules/no-unresolved.js:31:34)
    at checkSourceValue (file:///home/jailuser/git/node_modules/.pnpm/eslint-plugin-import-x@4.16.1_@typescript-eslint+utils@8.56.0_eslint@9.39.1_jiti@2.6.1__ef4159fe4f412a19f929f063c6fe191d/node_modules/eslint-plugin-import-x/lib/utils/module-visitor.js:14:9)
    at checkSource (file:///home/jailuser/git/node_modules/.pnpm/eslint-plugin-import-x@4.16.1_@typescript-eslint+utils@8.56.0_eslint@9.39.1_jiti@2.6.1__ef4159fe4f412a19f929f063c6fe191d/node_modules/eslint-plugin-import-x/lib/utils/module-visitor.js:17:9)
    at ruleErrorHandler (/home/jailuser/git/node_modules/.pnpm/eslint@9.39.1_jiti@2.6.1/node_modules/eslint/lib/linter/linter.js:1173:33)
    at /home/jailuser/git/node_modules/.pnpm/eslint@9.39.1_jiti@2.6.1/node_modules/eslint/lib/linter/source-code-visitor.js:76:46
    at Array.forEach (<anonymous>)
    at SourceCodeVisitor.callSync (/home/jailuser/git/node_modules/.pnpm/eslint@9.39.1_jiti@2.6.1/node_modules/eslint/lib/linter/source-code-visitor.js:76:30)
    at /home/jailuser/git/node_modules/.pnpm/eslint@9.39.1_jiti@2.6.1/node_modules/eslint/lib/linter/source-code-traverser.js:291:18
    at Array.forEach (<anonymous>)
    at SourceCodeTraverser.traverseSync (/home/jailuser/git/node_modules/.pnpm/eslint@9.39.1_jiti@2.6.1/node_modules/eslint/lib/linter/source-code-traverser.js:290:10)
    at runRules (/home/jailuser/git/node_modules/.pnpm/eslint@9.39.1_jiti@2.6.1/node_modules/eslint/lib/linter/linter.js:1214:12)
    at `#flatVerifyWithoutProcessors` (/home/jailuser/git/node_modules/.pnpm/eslint@9.39.1_jiti@2.6.1/node_modules/eslint/lib/linter/linter.js:2101:4)
    at Linter._verifyWithFlatConfigArrayAndWithoutProcessors (/home/jailuser/git/node_modules/.pnpm/eslint@9.39.1_jiti@2.6.1/node_modules/eslint/lib/linter/linter.js:2189:43)
    at Linter._verifyWithFlatConfigArray (/home/jailuser/git/node_modules/.pnpm/eslint@9.39.1_jiti@2.6.1/node_modules/eslint/lib/linter/linter.js:2292:15)
    at Linter.verify (/home/jailuser/git/node_modules/.pnpm/eslint@9.39.1_jiti@2.6.1/node_modules/eslint/lib/linter/linter.js:1677:10)
    at Linter.verifyAndFix (/home/jailuser/git/node_modules/.pnpm/eslint@9.39.1_jiti@2.6.1/node_modules/eslint/lib/linter/linter.js:2557:20)
    at verifyText (/home/jailuser/git/node_modules/.pnpm/eslint@9.39.1_jiti@2.6.1/node_modules/eslint/lib/eslint/eslint-helpers.js:1180:45)
    at readAndVerifyFile (/home/jailuser/git/node_modules/.pnpm/eslint@9.39.1_jiti@2.6.1/node_modules/eslint/lib/eslint/eslint-helpers.js:1321:10)

(import-x/no-unresolved)